cmd/auth/login.go

```diff
@@ -39,35 +58,30 @@ func newLoginCommand(persistentAuth *auth.PersistentAuth) *cobra.Command {
 
 	var loginTimeout time.Duration
 	var configureCluster bool
+	var profileName string
 	cmd.Flags().DurationVar(&loginTimeout, "timeout", auth.DefaultTimeout,
 		"Timeout for completing login challenge in the browser")
 	cmd.Flags().BoolVar(&configureCluster, "configure-cluster", false,
 		"Prompts to configure cluster")
+	cmd.Flags().StringVarP(&profileName, "profile", "p", "", `Name of the profile.`)
```

Comment:
Isn't this flag already defined at the top level?

Reply:
You are right. Removing the profile flag override here.